2025R2/dpf-operator-spec-25-r2/core-concepts/dpf-types.md

```diff
@@ -132,48 +132,133 @@ For cyclic simulation models, the ``cyclic_support`` holds information about the
 
 ### Filtering
 
+<a id="filtering"></a>
+
+When handling large amounts of simulation data, you can select a subset of the data and filter it down to the data of interest to the analysis.
+
+In the DPF ecosystem this is called ``scoping`` the data, and is managed with ``scoping`` entities.
```

Comment:
Minor, in pydpf APIs we call types in capital letters,  `my_scoping = dpf.Scoping()`. Then, it is true that we use lower-case letters in pin names and similar places. As here we are naming them, I'd use capital letters for consistency throughout this document

Reply:
Hi @rafacanton, this documentation is not for the PyDPF API.
On the contrary, the point is to document the framework, and the framework exposes these types through operator pins. This is the logic behind the use of ``scoping`` and such. This page in particular is what we redirect to when clicking on a pin accepted type in the operator documentation.